doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -193,4 +193,30 @@ For distributed parallel processing, you usually pass both parameters:
       version="3d",
       mode="solver",
       additional_arguments="-t16 -cnf=m1:8,m2:8",
-   )
\ No newline at end of file
+   )
+
+Using the PYFLUENT_ADDITIONAL_ARGUMENTS environment variable
+------------------------------------------------------------
+You can set additional arguments also using this environment variable.
+This can be useful when running on multiple platforms, avoiding the need to change the
+Python scripts being run.
+
+.. code:: bash
+
+   #!/bin/bash
+   #
+   # Activate your favorite Python environment
+   #
+   export AWP_ROOT232=/apps/ansys_inc/v232
+   . ./venv/bin/activate
+   #
+   # Set additional arguments
+   #
+   export PYFLUENT_ADDITIONAL_ARGUMENTS="-cnf=m1:8,m2:8 -t16 -peth.efa"
+   #
+   # Run a PyFluent script
+   #
+   python run.py
+
+Note: You should omit the additional_arguments option in the launch_fluent function as
+that variable is ignored when using the environmental variable.
```

Comment:
```suggestion
Note: You should omit the ``additional_arguments`` option in the ``launch_fluent`` function as
that variable is ignored when using the environment variable.
```
Can also remove this point as suggested by Mainak

Reply:
The note has been removed